api/reference/1.0/hosts/index.md

```diff
@@ -14,7 +14,7 @@ title: Overview
 1. `akamai`
 1. `sftp`
 
-When a `Build` is requested by the Launch user, Launch will check the `Library` to determine which `Environment` should build for.  The `Environment` has a relationship with the `Host` and that is how Launch knows which `Host` to use to deliver the `Build.
+When a `Build` is requested by the Launch user, Launch will check the `Library` to determine which `Environment` it should be built for.  The `Environment` has a relationship with the `Host` and that is how Launch knows which `Host` to use to deliver the `Build.
```

Comment:
When a `Build` is requested by the Launch user, Launch will check the `Library` to determine which `Environment` should build for.
->
`Environment` `the library should be built to.`

Reply:
I will adjust that to make it clearer that it's the _Library_ that's being built, not the _Build_ that's being built.